diff --git a/dist/runtime/extensions/accordion/index.mjs b/dist/runtime/extensions/accordion/index.mjs
index 292edcf085c90a04507ad0bdd6e7457e35ee1c44..ed2ffeba4315efc460f78b108c3c2f496a4513dd 100644
--- a/dist/runtime/extensions/accordion/index.mjs
+++ b/dist/runtime/extensions/accordion/index.mjs
@@ -1,2 +1,11 @@
-import node from "./node.mjs";
-export default { node };
+import Details from "@tiptap-pro/extension-details";
+export default {
+  node: Details.extend({
+    addAttributes() {
+      return {
+        ...this.parent?.(),
+        toggle: { default: true }
+      };
+    }
+  }).configure({ persist: true, HTMLAttributes: { class: "details" } })
+};
diff --git a/dist/runtime/extensions/accordion-content/index.mjs b/dist/runtime/extensions/accordion-content/index.mjs
index 292edcf085c90a04507ad0bdd6e7457e35ee1c44..067632d29bf44692a348afa89f9e3f27ddbc1626 100644
--- a/dist/runtime/extensions/accordion-content/index.mjs
+++ b/dist/runtime/extensions/accordion-content/index.mjs
@@ -1,2 +1,2 @@
-import node from "./node.mjs";
-export default { node };
+import DetailsContent from "@tiptap-pro/extension-details-content";
+export default { node: DetailsContent };
diff --git a/dist/runtime/extensions/accordion-summary/index.mjs b/dist/runtime/extensions/accordion-summary/index.mjs
index 292edcf085c90a04507ad0bdd6e7457e35ee1c44..40d853dcee8e961a8175feb92d6985d15e117610 100644
--- a/dist/runtime/extensions/accordion-summary/index.mjs
+++ b/dist/runtime/extensions/accordion-summary/index.mjs
@@ -1,2 +1,2 @@
-import node from "./node.mjs";
-export default { node };
+import DetailsSummary from "@tiptap-pro/extension-details-summary";
+export default { node: DetailsSummary };
